src/main/scala/xiangshan/frontend/FrontendBundle.scala

```diff
@@ -119,6 +119,8 @@ object ExceptionType {
   def af:    UInt = "b11".U // instruction access fault
   def width: Int  = 2
 
+  def hasException(e: UInt): Bool = e =/= none
```

Comment:
Great idea. Maybe replace the other uses of `e === ExceptionType.none` with `!ExceptionType.hasException(e)` by the way.